### ISSUE TITLE
HTML Element Plugin 0.0.2.7: HTML options "ID", "Class" and "Attribute" no longer seem to work.

### ISSUE THREAD

**Issue by MattouBatou** [el3um4s/construct-plugins-and-addons#4]
![c3_HTMLE_bug](https://user-images.githubusercontent.com/5024678/76950740-6d21bc80-6902-11ea-914f-e2c2586f1567.PNG)

None of the above options, apart from tag are applied.
![c3_HTMLE_bug_element](https://user-images.githubusercontent.com/5024678/76950907-ab1ee080-6902-11ea-9cf0-aff91e507af2.PNG)

**Comment by MattouBatou** [comment]
I figured this out. 
It was due to loading the layout with HTMLElement object in through an event, "On loader layout complete".

I presume this is too early to load something with HTMLElement objects since the engine still needs to load plugins perhaps?

If there's a prefered way to load in a layout with HTMLElement objects then it would be handy to know. I have a form I need to integrate into the end of a game. Maybe by that point though, the form will load just fine as the engine will have already loaded everything.

This might be worth adding to your documentation for anyone else falling into the same trap.

Thanks!
